README.md

```diff
@@ -254,6 +254,7 @@ Core rules are included in the `tslint` package.
 * `arrow-parens` requires parentheses around the parameters of arrow function definitions.
 * `ban` bans the use of specific functions. Options are ["object", "function"] pairs that ban the use of object.function().
 * `class-name` enforces PascalCased class and interface names.
+* `consecutive-function-overloads` enforces function overloads to be consecutive.
```

Comment:
how about naming this `adjacent-overload-signatures`?

make sure to name the walker class appropriately as well; in this case it would be `AdjacentOverloadSignaturesWalker`